cfgov/v1/templatetags/share.py

```diff
@@ -42,3 +42,8 @@ def v1page_permissions(context, page):
         )
 
     return context['user_page_permissions'].for_page(page)
+
+@register.filter
```

Comment:
I'd like for this to be somewhere else -- either in the block definition or its own templatetags file. It's also possible this is a filter that already exists? In any case, just wanted to point out that this is temporary